engine/installation/linux/ubuntulinux.md

```diff
@@ -59,13 +59,14 @@ packages from the Docker repository:
 4.  Add the new `GPG` key.
 
     ```bash
-    $ sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+    $ sudo apt-key adv \
+                  --keyserver hkp://ha.pool.sks-keyservers.net:80 \
+                  --recv-keys \
+                  58118E89F3A912897C070ADBF76221572C52609D
```

Comment:
can you move the key _after_ `--recv-keys` here? Make it clearer that it's a parameter for that flag, and not "output" of the command